src/components/Table/GridTable.test.tsx

```diff
@@ -634,6 +634,7 @@ describe("GridTable", () => {
           </div>
         );
       }
+
```

Comment:
this is some weird prettier formatting.. I haven't touched these lines.. I could remove